WalletWasabi/BitcoinCore/CoreNode.cs

```diff
@@ -130,6 +130,7 @@ public static async Task<CoreNode> CreateAsync(CoreNodeParams coreNodeParams, Ca
 				{
 					$"{configPrefix}.server			= 1",
 					$"{configPrefix}.listen			= 1",
+					$"{configPrefix}.daemon			= 0", // https://github.com/zkSNACKs/WalletWasabi/issues/3588
```

Comment:
I am not very familiar with the part of the code so, take my comment with a bit of salt. I think with should touch the config file as less as possible and for that reason, given the daemon option makes only sense for Linux and macOS, I would prefer to add that option conditionally. Something like this:

```c#
if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
{
		desiredConfigLines.Add($"{configPrefix}.deamon = 0");
}
```

Reply:
Daemon = 1 [ruins](https://old.reddit.com/r/WasabiWallet/comments/g654yt/wassabi_modifications_to_bitcoinconf_makes_port/foisr97/) bitcoind on Windows, so there's more good in it to add this option.